### PR TITLE
feat: [SONIC-350] Added 'Order Refunded' Segment event

### PR DESCRIPTION
**Pull request by shafqatfarhan**
**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, migration issues, deadlines, tickets

**Post-merge:**
- [ ] [Backported](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2065367719/Making+a+pull+request+for+a+named+release) to latest and next named releases

### CODE REVIEW

File: commerce_coordinator/apps/commercetools/views.py
Comment (aht007) [inline commerce_coordinator/apps/commercetools/views.py:1]: Are we adding the Refund event at the right place? The[ view name](https://github.com/edx/commerce-coordinator/pull/193/files?diff=split&w=0#diff-4a77126e8f58c013cbe2eea9be3f41c7dfa804c0f4cc4e751236c96e2dcf8c5eR181) and the docstring suggest that it is for Sanctioned order. Are we mixing Refund and Returned here?

File: commerce_coordinator/apps/commercetools/views.py [hunk 1]
```diff
@@ -233,4 +236,86 @@ def post(self, request):
...
+        return brand
+
+    @staticmethod
+    def _prepare_segment_event_properties(order):  # pragma no cover
+        return {
+            'track_plan_id': 19,
+            'trigger_source': 'server-side',
+            'order_id': order.id,
+            'checkout_id': order.cart.id,
+            'return_id': '',  # TODO: Set CT return ID here.
```
[the remark this answers is not in the document]
Comment (shafqatfarhan) [reply commerce_coordinator/apps/commercetools/views.py:300]: yes. https://2u-internal.atlassian.net/browse/SONIC-391

File: commerce_coordinator/apps/commercetools/views.py [hunk 2]
```diff
@@ -233,4 +236,86 @@ def post(self, request):
...
+            'total': order.taxed_price.total_gross.cent_amount / pow(
```
[the remark this answers is not in the document]
Comment (shafqatfarhan) [reply commerce_coordinator/apps/commercetools/views.py:301]: I will create a utility.

File: commerce_coordinator/apps/commercetools/views.py
Comment (shafqatfarhan) [reply commerce_coordinator/apps/commercetools/views.py:1]: ah. I see. doc string was not changed in a copy/paste from the above endpoint (i.e. sanctions). I will fix this.

File: commerce_coordinator/apps/commercetools/views.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (shafqatfarhan) [reply commerce_coordinator/apps/commercetools/views.py:301]: Due to its nature, I can't create a utility around this. Can you suggest something here? I can take care of this in a future iteration.

**Review by aht007 — APPROVED** [review]
LGTM

**Pull request merged**